src/mame/casio/rz1.cpp

```diff
@@ -136,6 +134,17 @@ void rz1_state::map(address_map &map)
 	map(0xe000, 0xe001).w(FUNC(rz1_state::leds_w));
 }
 
+void rz1_state::pg0_map(address_map& map)
+{
+	map(0x0000, 0x7fff).rom();
+	map(0x8000, 0x9fff).ram().share("sample1");
+	map(0xa000, 0xbfff).ram().share("sample2");
+}
+
+void rz1_state::pg1_map(address_map& map)
+{
+	map(0x0000, 0x7fff).rom();
+}
 
```

Comment:
Please add a blank line here (two blank lines at the start of a new major section).